resources/views/welcome.blade.php

```diff
@@ -78,6 +83,20 @@ class="absolute -top-48 right-0 h-[25rem] transform translate-x-1/2 translate-y-
                         Sponsors
                     </h2>
                 </div>
+
+                @if(!$allSponsors->isEmpty())
+                    <div class="flex flex-wrap justify-center gap-12 mt-16">
+                        @foreach($allSponsors as $sponsor)
+                            <div class="border border-2 @if ($sponsor->sponsorTier->name == 'golden' && $sponsor->is_approved) border-gold block
+                            @elseif ($sponsor->sponsorTier->name == 'silver' && $sponsor->is_approved) border-silver block
+                            @elseif ($sponsor->sponsorTier->name == 'bronze' && $sponsor->is_approved) border-bronze hidden xl:block @endif rounded-lg">
+                                <a href="{{ $sponsor->website }}">
+                                    <img alt="{{ $sponsor->name }}" src="{{ $sponsor->logo_path }}" class="w-full h-full px-6 py-6">
```

Comment:
Do the logos complete fill up the block or do they have 'whitespace' around them? I think its better if the boxes are the same size as the image/logo. Looks more professional.